RELEASE_PROCESS.md

```diff
@@ -24,33 +24,71 @@ In order to release, you will need the following accounts/permissions:
 - Chef Software, Inc Slack account
 - blog.chef.io wordpress account
```

Comment:
you don't need a blog post account now, you now need a discourse account set up with your chef.io email address so discourse automatically adds you to the Chef staff group.

https://www.chef.io/blog/2016/03/29/moving-release-and-security-announcements-to-discourse/